fix_sde_libs.sh

```diff
@@ -7,7 +7,7 @@ then
     exit 1
 fi
 SDE_INSTALL_PATH=$1
-la_files=(./install/lib/*.la)
+la_files=($1/install/lib/*.la)
```

Comment:
`"$SDE_INSTALL_PATH/install/lib/*.la"`

Reply:
yes that's the expected location for la files, earlier it was incorrect